src/main/webapp/app/shared/modal/ModifyCancerTypeModal.tsx

```diff
@@ -0,0 +1,180 @@
+
+  function getCancerTypeFromCancerTypeSelectOption(cancerTypeSelectOption: CancerTypeSelectOption) {
+    const cancerType = new CancerType();
+    cancerType.code = cancerTypeSelectOption.code;
+    cancerType.mainType = cancerTypeSelectOption.mainType;
+    cancerType.subtype = cancerTypeSelectOption.subtype;
+    return cancerType;
+  }
+
+  async function getICancerTypeFromCancerType(cancerType: CancerType) {
+    try {
+      const iCancerType = await searchCancerTypes({ query: cancerType.subtype ? cancerType.subtype : cancerType.mainType });
+      return iCancerType['data'][0];
+    } catch {
+      return null;
+    }
+  }
+
+  useEffect(() => {
+    async function setIncludedCancerTypes() {
+      const promises = [];
+      for (const cancerType of includedCancerTypes) {
+        promises.push(getICancerTypeFromCancerType(cancerType));
+      }
+      setIncluded(
+        (await Promise.all<ICancerType>(promises)).reduce<CancerTypeSelectOption[]>((accumulator, cancerType) => {
+          if (!cancerType) {
+            return accumulator;
+          }
+
+          const option: CancerTypeSelectOption = {
+            label: getCancerTypeName(cancerType),
+            value: cancerType.id,
+            code: cancerType.code,
+            mainType: cancerType.mainType,
+            subtype: cancerType.subtype,
+          };
+          return [...accumulator, option];
+        }, [])
+      );
+    }
+
+    setIncludedCancerTypes();
+  }, [includedCancerTypes]);
+
+  useEffect(() => {
+    async function setExcludedCancerTypes() {
+      const promises = [];
+      for (const cancerType of excludedCancerTypes) {
+        promises.push(getICancerTypeFromCancerType(cancerType));
+      }
+      setExcluded(
+        (await Promise.all<ICancerType>(promises)).reduce<CancerTypeSelectOption[]>((accumulator, cancerType) => {
+          if (!cancerType) {
+            return accumulator;
+          }
+
+          const option: CancerTypeSelectOption = {
+            label: getCancerTypeName(cancerType),
+            value: cancerType.id,
+            code: cancerType.code,
+            mainType: cancerType.mainType,
+            subtype: cancerType.subtype,
+          };
+          return [...accumulator, option];
+        }, [])
+      );
+    }
+
+    setExcludedCancerTypes();
+  }, [excludedCancerTypes]);
+
+  const confirmDisabled = useMemo(() => included.length < 1, [included]);
+
+  return (
+    <SimpleConfirmModal
+      show={true}
+      title="Modify Cancer Type(s)"
+      onCancel={onCancel}
+      confirmDisabled={confirmDisabled}
+      onConfirm={() => {
+        onConfirm(
+          included.map(option => getCancerTypeFromCancerTypeSelectOption(option)),
+          excluded.map(option => getCancerTypeFromCancerTypeSelectOption(option))
+        );
+      }}
+      body={
+        <Container className="px-0">
+          <Row className="my-2">
+            <Col>
```

Comment:
Do we need to use container/row/col if you are not using the grids?

Reply:
Oh yeah, no need anymore since we removed horizontally aligned elements.